src/main/java/ie/ait/bteam/drcare/DrCareApplication.java

```diff
@@ -3,18 +3,21 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.web.servlet.ServletComponentScan;
 import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
 
 @SpringBootApplication
-public class DrCareApplication extends SpringBootServletInitializer {
+@ServletComponentScan
+public class DrCareApplication{
+//public class DrCareApplication extends SpringBootServletInitializer {
 
 	public static void main(String[] args) {
 		SpringApplication.run(DrCareApplication.class, args);
 	}
 
-	@Override
```

Comment:
after adding back the extends, uncomment this.

Reply:
Uncommented